index.js

```diff
@@ -1,79 +1,180 @@
+addStudentToClass("Alice");    // Adds Alice to the class
+addStudentToClass("");         // Logs: Cannot add an empty string as a student name.
+addStudentToClass("Alice");    // Logs: Student Alice is already in the class
+addStudentToClass("John");
+addStudentToClass("Jane");
+addStudentToClass("Mariya");
+addStudentToClass("Jerry");    // Logs: Cannot add more students to class 07
+addStudentToClass("Queen");    // Always adds Queen even if the class is full
+
+
+ console.log(getNumberOfStudents());
   
-  // Julia's house details
-  const juliaWidth = 5;
-  const juliaDepth = 11;
-  const juliaHeight = 8;
-  const juliaGardenSize = 70;
-  const juliaActualCost = 1000000;
-  
-  const juliaCalculatedPrice = calculateHousePrice(juliaWidth, juliaDepth, juliaHeight, juliaGardenSize);
-  console.log("Julia's calculated house price: " + juliaCalculatedPrice);
-  console.log("Julia's actual house cost: " + juliaActualCost);
-  if (juliaCalculatedPrice > juliaActualCost) {
-    console.log("Julia is paying less than the calculated price. Good deal!");
-  } else {
-    console.log("Julia is paying more than the calculated price. Not a good deal.");
+//Candy helper optional
+
+let boughtCandyPrices=[]
+function pricePerGramOfCandy(candyType) {
+  switch (candyType) {
+      case "sweet":
+          return 0.5;
+      case "chocolate":
+          return 0.7;
+      case "toffee":
+          return 1.1;
+      case "chewing-gum":
+          return 0.03;
+      default:
+          console.log("Unknown candy type");
+          return 0; // Return 0 for unknown candy types to avoid incorrect calculations
   }
-  //task 4
-// Step 1: Create the arrays
-const firstWords = ["Easy", "Awesome", "Corporate", "Creative", "Innovative", "Dynamic", "Smart", "NextGen", "Tech", "Green"];
-const secondWords = ["Solutions", "Enterprises", "Ventures", "Technologies", "Networks", "Systems", "Hub", "Labs", "Dynamics", "Corporation"];
+}
 
-// Step 2: Generate random indices
-const randomIndex1 = Math.floor(Math.random() * 10);
-const randomIndex2 = Math.floor(Math.random() * 10);
 
-// Step 3: Create the startup name
-const startupName = firstWords[randomIndex1] + " " + secondWords[randomIndex2];
+function addCandy(candyType,weight){
+  let pricePerGram = pricePerGramOfCandy(candyType);
+  boughtCandyPrices.push(pricePerGram * weight)
+ }
+addCandy("sweet",20);
+console.log(boughtCandyPrices);
 
-// Step 4: Log the startup name and its character count
-console.log("The startup: \"" + startupName + "\" contains " + startupName.length + " characters.");
-  
+let amountToSpend=Math.random() * 100;
+function canBuyMoreCandy(){
+  let totalSpent = 0;
+
+  for (let i = 0; i < boughtCandyPrices.length; i++) {
+      //totalSpent += boughtCandyPrices[i];
+  }
+
+  if (totalSpent < amountToSpend) {
+      //console.log("You can buy more, so please do!");
+      //return true;
+  } else {
+      //console.log("Enough candy for you!");
+      //return false;
```

Comment:
You probably forgot to uncomment these...

Reply:
yes i forgot to comment